CMakeLists.txt

```diff
@@ -336,15 +336,19 @@ set(GINKGO_TEST_INSTALL_SRC_DIR "${Ginkgo_SOURCE_DIR}/test/test_install/")
 set(GINKGO_TEST_INSTALL_BIN_DIR "${Ginkgo_BINARY_DIR}/test/test_install/")
 set(GINKGO_TEST_EXPORTBUILD_SRC_DIR "${Ginkgo_SOURCE_DIR}/test/test_exportbuild/")
 set(GINKGO_TEST_EXPORTBUILD_BIN_DIR "${Ginkgo_BINARY_DIR}/test/test_exportbuild/")
+set(GINKGO_TEST_EXPORTBUILD_PKGCONFIG_SRC_DIR "${Ginkgo_SOURCE_DIR}/test/test_exportbuild_pkgconfig/")
+set(GINKGO_TEST_EXPORTBUILD_PKGCONFIG_BIN_DIR "${Ginkgo_BINARY_DIR}/test/test_exportbuild_pkgconfig/")
 if(MSVC)
```

Comment:
nit: This should probably use [GENERATOR_IS_MULTI_CONFIG](https://cmake.org/cmake/help/v3.24/prop_gbl/GENERATOR_IS_MULTI_CONFIG.html) instead of only supporting MSVC